### ISSUE TITLE
Add meta/control namespace for online adding/removing/enabling of host properties, elements, etc..

### ISSUE THREAD

**Issue by claudiocabral** [saunaklub/etudes#16]
I think we should be able to create and destroy elements with OSC messages.
This should be  fairly easy to implement in the current state of the project,
I'll look into it this week.

**Comment by drlight-code** [comment]
Good point. To implement it, create a dedicated branch so can to do some reviewing together before it is merged to master. Also we have to decide on a kind of naming scheme for the messages which do not relate to etude or element inputs. We could think e.g. about having reserved namespaces like /control for host-level control messages, like switching between etudes or setting other global parameters, and /<etude>/control for etude-level control messages like creation of elements. Usage of those reserved names for actual inputs should then be catched and considered an error in the respective registerInputs. Also when parsing the config file and creating the etudes (there may be no etude named 'control'), to avoid accidental and surprising name clashes.
